src/main/java/net/sf/jabref/JabRefPreferences.java

```diff
@@ -455,6 +458,9 @@ private JabRefPreferences() {
         defaults.put(USE_PROXY, Boolean.FALSE);
         defaults.put(PROXY_HOSTNAME, "my proxy host");
         defaults.put(PROXY_PORT, "my proxy port");
+        defaults.put(USE_PROXY_AUTHENTICATION, Boolean.FALSE);
+        //defaults.put(PROXY_USERNAME, "my proxy username");
+        //defaults.put(PROXY_PASSWORD, "my proxy password");
```

Comment:
Because I followed the defaults for hostname and port I didn't get the idea behind populating it with "my proxy ...".